endline-script.py

```diff
@@ -0,0 +1,20 @@
+import os
+import subprocess
+
+cwdn = os.getcwd()
+excludepath = '! -path "./.*" ! -path "*.pyc" ! -path "*__.py" ! -path "*.svg" ! -path "*.png" '
+excludepath = excludepath + '! -path "./openwisp_utils.egg-info/SOURCES.txt"'
+filelist = subprocess.check_output('find . -type f ' + excludepath, cwd=cwdn, shell=True)
+filelist = filelist.decode('utf-8')
+pathvar = ''
+for c in filelist:
+    if(c != '\n'):
+        pathvar = pathvar + c
+    else:
+        check = subprocess.check_output('tail -c 1 ' + pathvar, cwd=cwdn, shell=True)
+        if(check.decode('utf-8') != '\n'):
+            print(pathvar + " Dont have a blank line at the end")
```

Comment:
`pathvar + " doesn't have a blank line at the end of the file"`

Reply:
Ok thanks :) sorry for my bad English